system-tests/project-fixtures/react/src/App.cy.jsx

```diff
@@ -4,5 +4,5 @@ import { App } from './App'
 it('renders hello world', () => {
   cy.mount(<App />)
   // Click on the header here to ensure that the AUT is interactable. This ensures that the dev server overlay is not displaying
-  cy.get('h1').contains('Hello World').click()
+  cy.get('h1').contains('Hello World').should('have.css', 'background-color', 'rgb(255, 0, 0)').click()
```

Comment:
Nitpick, but can we add this as a separate test case here? The test case is called "renders hello world" but I think we should be explicit about what we're looking for with a separate test case. Otherwise it's not clear why the assertion is checking for that CSS, imo